### ISSUE TITLE
PyTorch implementation

### ISSUE THREAD

**Issue by ccc1711** [NVlabs/contact_graspnet#3]
Thank you for making the code public.

Is there possibility of presenting the PyTorch implementation of your code?

**Comment by 123zhen123** [comment]
Yes, thank you very much for your work, if you can provide the PyTorch version I will be very grateful

**Comment by MartinSmeyer** [comment]
afaik, there is no plan to provide an official pytorch version
